src/vs/workbench/contrib/positronConsole/browser/components/consoleInstanceInfoButton.tsx

```diff
@@ -56,8 +56,13 @@ export const ConsoleInstanceInfoButton = () => {
 		}
 
 		// Get the channels from the session.
-		const channels =
-			intersectionOutputChannels(await session.listOutputChannels());
+		let channels: LanguageRuntimeSessionChannel[] = []
+		try {
+			channels = intersectionOutputChannels(await session.listOutputChannels());
+		} catch (err) {
+			// If we fail to get the channels we can just ignore it
+			console.warn('Failed to get output channels', err);
```

Comment:
This should use the log service rather than `console.warn` (so it gets the correct filtering treatment, etc.)

Reply:
Added logging here. Since now we have our own output channel, I also added more logging information that can be useful.

https://github.com/posit-dev/positron/pull/7024/commits/cefd51e3c309e43256b41ea60d333d081fe446d8